src/cli/msig/index.ts

```diff
@@ -163,6 +165,66 @@ export default async function msig(args: string[]) {
                 newGovernanceAccounts
             );
 
+            if (test) await executeProposal(proposer, proposalName, proposalHash);
+        } else if (proposalType === 'deploy-contract') {
+            const contractName = args[3];
+
+            if (!contractName) {
+                throw new Error('Contract name must be provided for deploy-contract proposal');
+            }
+
+            const contractInfo = {
+                account: contractName,
+                contractDir: path.join(__dirname, `../../Tonomy-Contracts/contracts/${contractName}`),
+            };
+            const action = await deployContract(contractInfo, signer, { throughTonomyProxy: true, actionData: true });
+
+            if (action) {
+                const proposalHash = await createProposal(
+                    proposer,
+                    proposalName,
+                    [action],
+                    privateKey,
+                    newGovernanceAccounts
+                );
+
+                if (test) await executeProposal(proposer, proposalName, proposalHash);
+            }
+        } else if (proposalType === 'eosio.code-permission') {
+            const ownerAuthority = Authority.fromAccount({ actor: 'vesting.tmy', permission: 'owner' });
+
+            ownerAuthority.addCodePermission('vesting.tmy');
+            const action = {
+                account: 'tonomy',
+                name: 'updateauth',
+                authorization: [
+                    {
+                        actor: 'tonomy',
+                        permission: 'active',
+                    },
+                    {
+                        actor: 'tonomy',
+                        permission: 'owner',
+                    },
+                ],
+                data: {
+                    account: 'found.tmy',
```

Comment:
Right now this only update one account. Instead it should update all of the accounts found in the `addCodePermissionsTo` array. https://github.com/Tonomy-Foundation/Tonomy-ID-SDK/blob/feature/update-cli-reploy-contracts/src/cli/bootstrap/index.ts#L211